assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+            "draw": [
+                ["LANCE*", "FUBOU", "SNOOK", "TOPAZ", "PERID"]
+            ]
+        },
+        "LANCEB": {
+            "icao": "LANCEB",
+            "name": "Lance East Bravo",
+            "entryPoints": {
+                "LANCE": [["LANCE", "A110+"]]
+            },
+            "body": [["FUBOU", "A60+"], ["SNOOK", "A40+"]],
+            "rwy": {
+                "RJSS27": ["#085"]
+            },
+            "draw": [
+                ["LANCE*", "FUBOU", "SNOOK"]
+            ]
+        },
+        "LANCEW": {
+            "icao": "LANCEW",
+            "name": "Lance West",
+            "entryPoints": {
+                "OWLET": [["LANCE", "A110+"]]
+            },
+            "body": [["QUAIL", "S250-"], ["RIBON", "A60+"]],
+            "rwy": {
+                "RJSS09": [["SHIPS", "A40+"]]
+            },
+            "draw": [
+                ["LANCE*", "QUAIL", "RIBON", "SHIPS"]
+            ]
+        },
+        "OWLEEA": {
+            "icao": "OWLEEA",
+            "name": "Owlet East Alfa",
+            "entryPoints": {
+                "OWLET": [["OWLET", "A130+"]]
+            },
+            "body": [["DATTE", "A80+"], ["RIBON", "A60+"], ["SNOOK", "A40+"], ["TOPAZ", "S230-"], ["PERID", "A15+"]],
+            "rwy": {
+                "RJSS27": ["#360"]
+            },
+            "draw": [
+                ["OWLET*", "DATTE", "RIBON", "SNOOK", "TOPAZ", "PERID"]
+            ]
+        },
+        "OWLEEB": {
+            "icao": "OWLEEB",
+            "name": "Owlet East Bravo",
+            "entryPoints": {
+                "OWLET": [["OWLET", "A130+"]]
+            },
+            "body": [["DATTE", "A80+"], ["RIBON", "A60+"], ["SNOOK", "A40+"]],
+            "rwy": {
+                "RJSS27": ["#067"]
+            },
+            "draw": [
+                ["OWLET*", "DATTE", "RIBON", "SNOOK"]
+            ]
+        },
+        "OWLETW": {
+            "icao": "OWLETW",
+            "name": "Owlet West",
+            "entryPoints": {
+                "OWLET": [["OWLET", "A130+"]]
+            },
+            "body": [["PRINK", "A130+"], ["QUIST", "S250-"], ["RIBON", "A60+"]],
+            "rwy": {
+                "RJSS09": [["SHIPS", "A40+"]]
+            },
+            "draw": [
+                ["OWLET*", "PRINK", "QUIST", "RIBON", "SHIPS"]
+            ]
+        },
+        "PERID": {
+            "icao": "PERID",
+            "name": "Perid",
+            "entryPoints": {
+                "AOBAR": ["AOBAR"]
+            },
+            "body": ["_SDE135010", "_SDE120010", "_SDE105010"],
+            "rwy": {
+                "RJSS27": ["PERID"]
+            },
+            "draw": [
+                ["AOBAR*", "_SDE135010", "_SDE120010", "_SDE105010", "PERID"]
+            ]
+        }
+    },
+    "spawnPatterns": [
+        {
+            "origin": "RJSS",
+            "destination": "",
+            "category": "departure",
+            "route": "RJSS27.STEED3.RIKYU",
```

Comment:
Although not necessary, it would be good to extend the route further away from the airspace. Same for the other departures too

Reply:
How far in km/nm would you suggest them be away?

Reply:
Roughly the same distance as the start of the arrival spawnPatterns

Reply:
Okay, thanks